internal/command/meta_backend_migrate.go

```diff
@@ -21,14 +21,14 @@ import (
 )
 
 type backendMigrateOpts struct {
-	OneType, TwoType string
-	One, Two         backend.Backend
+	SourceType, DestinationType string
+	Source, Destination         backend.Backend
 
 	// Fields below are set internally when migrate is called
 
-	oneEnv string // source env
-	twoEnv string // dest env
-	force  bool   // if true, won't ask for confirmation
+	sourceEnv      string
+	destinationEnv string
```

Comment:
I think we should rename these to `sourceWorkspace` and `destinationWorkspace`. ("Environment" is the prior name for a CLI "workspace".)

Reply:
Will do